### DISCUSSION TITLE
直播互动，教程素材收集 20230418

### DISCUSSION THREAD

**Discussion by ks-bao** [istoreos/istoreos#721] [General]
通过直播来收集教程素材信息，并且也录制一些磁盘挂载相关的教程素材。
有其他更好的建议，可以在本贴回复。

* 固件放一个工单的入口
* 软件中心放一个帮助入口
* 系统首页放一个官方文档入口
* 文件左边可以自定义文件夹，或者挂载网络的文件夹

**Comment by felixwann** [comment]
我的设备是联想M73+i3 4360T+4G DDR3 1333Mhz
我用iStoreOS，在一台设备测网速的时候，其他设备会短暂断网，直至测速结束网络才恢复；
但是用爱快就没有出现断网情况。重装iStoreOS也一样，请问是哪个地方需要配置吗？

**Comment by xviivx88** [comment]
我有一个小小的建议。有好用的iStoreOS+易有云组的个人云，11块硬盘，3个RAID5，1个RAID0，目前想知道某一块或者所有硬盘的健康度不知道在哪里看。能否增加这么一个功能，展示出所有硬盘的一个健康信息。谢谢

**Reply by jannson** [reply]
我们有 S.M.A.R.T 检测，我们研究下，看看能否有个更便捷的方式来看到这个健康信息。

**Comment by xviivx88** [comment]
突发奇想到一个建议，主界面有没有可能做成可自定义的。由用户来选择那些信息模块显示在主页。当然有一个默认的布局，然后固定1到2个模块显示。其他模块用户自定义是否显示。

**Reply by jannson** [reply]
嗯，你这个想法是不错的。当初的设计确实这样去思考过。

**Comment by dnpl** [comment]
首先可以加上IPv6 DNS显示。

**Reply by jannson** [reply]
我们考虑下，看看有没有地方可以放。
